pkg/cmd/server/api/validation/master.go

```diff
@@ -654,7 +655,12 @@ func ValidateAPIServerExtendedArguments(config api.ExtendedArguments, fldPath *f
 }
 
 func ValidateControllerExtendedArguments(config api.ExtendedArguments, fldPath *field.Path) field.ErrorList {
-	return ValidateExtendedArguments(config, controlleroptions.NewCMServer().AddFlags, fldPath)
+	addFlags := func(fs *pflag.FlagSet) {
```

Comment:
Compare with https://github.com/sttts/origin/pull/22.

Am fine with this for the time being. Will rebase my change onto this one.